### PR TITLE
Add basic Github issue template

### PR DESCRIPTION
**Pull request by timss**

### CODE REVIEW

**Comment by timss** [comment]
@gkatsev looks ok?

**Comment by SammysHP** [comment]
I'd ask for the output of `:version` (which can be extended if necessary). Also I suggest to include a minimal structure for issues (e.g. trigger, expected behavior, steps to reproduce).

--------

##### Output of `:version`
```
Paste the output here
```

##### Description

##### Expected behavior

##### Steps to reproduce
1. 
1. 
1. 

--------

    ##### Output of `:version`
    ```
    Paste the output here
    ```
    
    ##### Description
    
    ##### Expected behavior
    
    ##### Steps to reproduce
    1. 
    1. 
    1.

**Comment by timss** [comment]
Example issue, as inspired by #435:

---

##### Issue type:

- Bug report

##### Version:

```
Vimperator: 3.12.0 (created: 2016/02/28 22:07:56)
Firefox:    Mozilla/5.0 (X11; Linux x86_64; rv:44.0) Gecko/20100101 Firefox/44.0 Iceweasel/44.0.2
```

##### Description:

Closing tab by pressing `d` does not return to previous tab.

##### Expected behavior:

Pressing `d` should do the same thing as clicking the `(X)` button.

##### Steps to reproduce:

1. Open a new instance of Firefox
2. Press Ctrl-T to open a new tab (there should now be two tabs)
3. Switch back to the first tab
4. Press Ctrl-T to open a third tab (you should be on the third tab)

---

https://github.com/vimperator/vimperator-labs/pull/450#issuecomment-194969359

**Comment by timss** [comment]
Note: while new commits to a PR could've been `git push -f`ed, I think it's better to rebase/squash it on merge instead.

**Comment by SammysHP** [comment]
"Issue type" should be set by a project member as a label. If you think that the user should provide it during creation nevertheless, then I suggest checkboxes like

----

##### Issue type:

- [x] Bug report
- [ ] Feature request
- [ ] Version compatibility

----

and remove that section after adding the labels.

**Comment by timss** [comment]
The intention behind 'issue type' is just to ease the work of maintainers, even if labels are to be added.

I believe checkboxes are in fact a task list, like in #168, and I think it'll show as '1 of 3 tasks complete'.

Aren't the checkboxes in fact a task list, like in #168 and referenced in the top of this issue? Wouldn't it show as '1 of 3 tasks complete' if it is to be referenced later as per your example?

**Comment by SammysHP** [comment]
You're right, it will be shown as a task list. :/

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+##### Version:
+
+```
+<!-- (Paste the output of `:version` here) -->
+```
+
+##### Description:
+
+<!-- Explain the issue briefly -->
```
Comment (gkatsev) [inline ISSUE_TEMPLATE.md]: s/briefly/in great detail/
:)

Comment (timss) [inline ISSUE_TEMPLATE.md]: Fixed.

**Comment by timss** [comment]
Great, thanks :D

**Pull request merged**